LongestPalindrome.java

```diff
@@ -0,0 +1,40 @@
+package day6;
+
+import java.util.*;
+//Time Complexity : O(n) where n is length of s
+// Space Complexity : O(1) [space required by count variable]
```

Comment:
What about space by HashSet ?

Reply:
yes, O(n) for iterating over the array and storing elements in Hashset.